src/test/java/github/weichware10/analyse/DiagrammTest.java

```diff
@@ -39,18 +39,18 @@ public void createRelFreqImgAreaShouldWork() {
      */
     @Test
     public void createViewTimeDistrSouldWork() {
-        Diagramm dia1 = new Diagramm(new Data(ToolType.ZOOMMAPS, 1), new ConfigDiagramm());
+        Diagramm dia1 = new Diagramm(new TrialData(ToolType.ZOOMMAPS, "1", "3"), new ConfigDiagramm());
         assertEquals("createViewTimeDistr sollte .../diagramm/VIEWTIMEDISTR_ZOOMMAPS_1 zurückgeben",
                 ".../diagramm/VIEWTIMEDISTR_ZOOMMAPS_1.jpg",
                 dia1.createViewTimeDistr());
 
-        Diagramm dia2 = new Diagramm(new Data(ToolType.EYETRACKING, 5), new ConfigDiagramm());
+        Diagramm dia2 = new Diagramm(new TrialData(ToolType.EYETRACKING, "5", "3"), new ConfigDiagramm());
         assertEquals(
                 "createViewTimeDistr sollte .../diagramm/VIEWTIMEDISTR_EYETRACKING_5 zurückgeben",
                 ".../diagramm/VIEWTIMEDISTR_EYETRACKING_5.jpg",
                 dia2.createViewTimeDistr());
 
-        Diagramm dia3 = new Diagramm(new Data(ToolType.CODECHARTS, 132), new ConfigDiagramm());
+        Diagramm dia3 = new Diagramm(new TrialData(ToolType.CODECHARTS, "132", "3"), new ConfigDiagramm());
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.sizes.LineLengthCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line is longer than 100 characters (found 107).